ionoscloud/resource_cube_server.go

```diff
@@ -47,7 +47,8 @@ func resourceCubeServer() *schema.Resource {
 			"hostname": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				Description:      "The hostname of the resource. Allowed characters are a-z, 0-9 and - (minus). Hostname should not start with minus and should not be longer than 63 characters.",
+				Computed:         true,
+				Description:      "The hostname of the resource. Allowed characters are a-z, 0-9 and - (minus). Hostname should not start with minus and should not be longer than 63 characters. If not value give, it will receive the name of the server",
```

Comment:
@cristiGuranIonos same here: `if not value given` or `if no value is given/provided`